### PR TITLE
config: fix email sending bug

### LINKED ISSUE #28: sign up: error when the user tries to create a new account
Way to reproduce, click signup to app-ilis, fill the form up and click the "Sign up" button

```
Traceback (most recent call last):
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/werkzeug/wsgi.py", line 660, in __call__
    return app(environ, start_response)
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/flask_security/decorators.py", line 230, in wrapper
    return f(*args, **kwargs)
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/flask_security/views.py", line 118, in register
    user = register_user(**form.to_dict())
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/flask_security/registerable.py", line 41, in register_user
    'welcome', user=user, confirmation_link=confirmation_link)
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/flask_security/utils.py", line 401, in send_mail
    mail.send(msg)
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/flask_mail.py", line 492, in send
    message.send(connection)
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/flask_mail.py", line 427, in send
    connection.send(self)
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/flask_mail.py", line 194, in send
    email_dispatched.send(message, app=current_app._get_current_object())
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/blinker/base.py", line 267, in send
    for receiver in self.receivers_for(sender)]
  File "/Users/user/.virtualenvs/invenio-first-week/src/invenio-mail/invenio_mail/ext.py", line 47, in print_email
    '{0}\n{1}\n'.format(message.as_string(), '-' * 79))
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/flask_mail.py", line 381, in as_string
    return self._message().as_string()
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/flask_mail.py", line 330, in _message
    msg['From'] = sanitize_address(self.sender, encoding)
  File "/Users/user/.virtualenvs/invenio-first-week/lib/python2.7/site-packages/flask_mail.py", line 105, in sanitize_address
    nm, addr = addr
ValueError: too many values to unpack

```

### PR DESCRIPTION
**Pull request by switowski**
* Fixes the problem with flask-security (#685) failing to send an email
  (closes #28).

### CODE REVIEW

**Review by drjova — APPROVED** [review]

File: invenio_app_ils/ui/backoffice/src/pages/ItemDetails/ItemDetails.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      actionLoading,
+      itemActionError,
+    } = this.props;
+    if (fetchLoading) return <Loader active inline="centered" />;
+    if (itemActionError) {
+      console.error(data);
+      return (
+        <Message
+          icon="exclamation"
+          header="Oups, something went horribly wrong!"
```
Comment (ntarocco) [inline invenio_app_ils/ui/backoffice/src/pages/ItemDetails/ItemDetails.js:32]: Well.. we need to remember to check all these error messages...

File: invenio_app_ils/ui/backoffice/src/pages/ItemDetails/state/actions.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  SET_ITEM_FETCH_LOADING,
+  SET_ITEM_FETCH_DETAILS,
+  SET_ITEM_ACTION_LOADING,
+  SET_ITEM_ACTION,
+  SET_ITEM_ACTION_ERROR,
+} from './types';
+import { fetchItem, postItem } from './api';
+import { serializeItemDetails } from './selectors';
+
+export const fetchItemDetails = itemid => {
```
Comment (ntarocco) [inline invenio_app_ils/ui/backoffice/src/pages/ItemDetails/state/actions.js:4]: @topless @zzacharo given that we have defined the function like this:
`fetchItemDetails` = params =>

it should already set the correct context for `this`, so we should not need the `bind(this)` in the constructor of the component.
WDYT?

File: invenio_app_ils/ui/backoffice/src/pages/ItemDetails/components/ItemActions/ItemActions.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            {action}
+          </Button>
+        </List.Item>
+      );
+    });
+  }
+
+  render() {
+    let { actions, data, actionLoading } = this.props;
+    if (actionLoading) return <Loader active inline="centered" />;
```
Comment (ntarocco) [inline invenio_app_ils/ui/backoffice/src/pages/ItemDetails/components/ItemActions/ItemActions.js]: In general I am not super fan of multiple exit points (multiple `return`) in a function, but I can survive with that :)

File: invenio_app_ils/ui/backoffice/src/pages/ItemDetails/state/api.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import axios from 'axios';
```
Comment (ntarocco) [inline invenio_app_ils/ui/backoffice/src/pages/ItemDetails/state/api.js:1]: For this, we should use `axios` `instance` and request intercept, but we can do in a second moment.
See:
* https://github.com/axios/axios#creating-an-instance
* https://github.com/axios/axios#interceptors

File: invenio_app_ils/ui/backoffice/src/pages/ItemDetails/state/actions.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  return async dispatch => {
+    dispatch({
+      type: SET_ITEM_FETCH_LOADING,
+      payload: {},
+    });
+
+    let details = await fetchItem(itemid).catch(reason => {
+      dispatch({
+        type: SET_ITEM_ACTION_ERROR,
```
Comment (ntarocco) [inline invenio_app_ils/ui/backoffice/src/pages/ItemDetails/state/actions.js]: it should be `SET_ITEM_FETCH_LOADING`.

File: invenio_app_ils/ui/backoffice/src/pages/ItemDetails/state/actions.js (hunk 2 above)
Comment (zzacharo) [reply invenio_app_ils/ui/backoffice/src/pages/ItemDetails/state/actions.js:4]: It is but there were some differences in the 2 ways but more in the way that the method is attached to either the instance or the prototype. But if in theory don't use inheritance, which is not suggested, then is fine and we can transform everything in arrow functions :)

**Pull request merged**